pull_query.py

```diff
@@ -30,6 +21,6 @@ def pull_query(config_path: str):
     try:
         tweet_query.pull(query, output_dir=str(config.local.output_dir),
             start_time = config.local.start_time, end_time = config.local.end_time,
-            max_results = config.local.max_query_response)
+            max_results = config.local.max_query_response, batch_size  = config.twitter.account.tweets_per_query)
```

Comment:
Should batch size be dropped?

Reply:
It could be dropped and instead be replaced with the maximum (500) when using the Tweepy search_all_tweets, but it is serving a different purpose than max_results, as noted in other comments.

Reply:
Agreed